eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -487,11 +487,10 @@ asset_name = bytes .size (0..32)
 negInt64 = -9223372036854775808 .. -1
 posInt64 = 1 .. 9223372036854775807
 nonZeroInt64 = negInt64 / posInt64 ; this is the same as the current int64 definition but without zero
-natNum = 1 .. 4294967295 ; non-zero 32bit uint
 
 positive_coin = 1 .. 18446744073709551615
 
-value = positive_coin / [positive_coin,multiasset<natNum>]
+value = positive_coin / [positive_coin,multiasset<positive_coin>]
```

Comment:
Needs confirming that `uint64` is actually allowed for multiasset in a value. Same code seems to be used to parse multi-assets in the mint and in values, and mint allows type `int64`, because it can be negative. There is a suspicion that multiasset in values can also only have the 9223372036854775807 maximum and not the 18446744073709551615

Reply:
Although this line does suggest that asset amounts are supposed to be Word64 as well, so should be all good if it's really so: https://github.com/input-output-hk/cardano-ledger/blob/c1603995da7c57d95371844191d235b05bb476b7/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L494

Reply:
And if those Word64a are treated is `uint64` and not as `int64`

Reply:
for `Value` (which is used it transaction outputs), you can see from the code snippets in the PR notes that the Haskell is decoding a `Word64` and producing an error for zero.

```
$ ghci
GHCi, version 8.8.4: https://www.haskell.org/ghc/  :? for help
Prelude> import Data.Word
Prelude Data.Word> maxBound :: Word64
18446744073709551615
```

Reply:
The PR is all correct, I have figured finally that `decodePositiveCoin` is used as the quantities decoder, when `decodeMultiAssetMaps` is called for a value, while a different `decodeIntegerBounded64` function is used as a decoder, when parsing a mint. So everything is correct and the `positive_coin` type is right to be used for a value multiasset 👍